ethereum_history_api/oracles/src/noir/oracles/transactionOracle/encode.ts

```diff
@@ -9,10 +9,13 @@ import { removeHexPrefix } from '../../../util/hex.js';
 
 const MAX_TX_KEY_LEN = 3;
 export const MAX_TX_KEY_NIBBLE_LEN = 6;
+export const MAX_TX_PREFIXED_KEY_NIBBLE_LEN = MAX_TX_KEY_NIBBLE_LEN + BYTE_HEX_LEN;
 export const MAX_TX_TREE_DEPTH = MAX_TX_KEY_NIBBLE_LEN + 1;
 export const MAX_TX_ENCODED_LEN = 525;
 export const MAX_TX_RLP_LEN = MAX_TX_ENCODED_LEN - 1;
 export const MAX_TX_PROOF_LEN = MAX_TRIE_NODE_LEN * MAX_TX_TREE_DEPTH;
+export const MAX_TX_SIZE_M = 1000;
+export const MAX_TX_LEAF_SIZE_M = 1043; // See calculations in receiptOracle/encode.ts
```

Comment:
Please copy those calculations here as eventually those would be different values